### PR TITLE
feat(graduateexplorer): add typdef, resolvers and service for geodata

### PR DESCRIPTION
**Pull request by hip1310**

### CODE REVIEW

File: src/graphql/graduateExplorer/service.js [hunk 1]
```diff
@@ -2,6 +2,22 @@ const _ = require('lodash')
 const { eCarmen } = require('../../lib/clients')
 const { getEcarmenPaginationObj } = require('../../lib/data/utils')
 
+const getGraduateExplorerGeoData = async ({ args, data, context }) => {
+  try {
+    const universityDimName = _.get(data, 'education.school.name.dim', '')
```
Comment (masiamj) [inline src/graphql/graduateExplorer/service.js]: If there's no `universityDimName` do we want to throw an error? The results probably won't make sense and we probably don't want to expose any data that would come back to anybody.

File: src/graphql/graduateExplorer/typeDefs.js [hunk 2]
```diff
@@ -148,11 +148,11 @@ const typeDefs = gql`
     meta: GraduateExplorerMeta! # This can be it's own resolver where we just use _.pick(parent, ['totalCount', 'mastersCount', ...others])
     skills: [Skill]!
     people(pagination: PaginationInput!): PersonStatistic!
-    # Geodata
   }
 
   extend type Query {
     graduateExplorer(data: ECarmenFilterInput): GraduateExplorerResult!
+    geoData(data: ECarmenFilterInput): JSON
```
Comment (masiamj) [inline src/graphql/graduateExplorer/typeDefs.js]: Should this be non-nullable `JSON!`?

File: src/lib/clients/eCarmen/index.js [hunk 3]
```diff
@@ -19,6 +19,14 @@ exports.getGraduateExplorerData = async (params) => {
   return {}
 }
 
+exports.getGraduateExplorerGeoData = async (body, universityDimName) => {
+  const result = await eCarmenClient.post(
+    `/core/v1/grad_explorer/geodata/${universityDimName.replace(' ', '%20')}`,
```
Comment (masiamj) [inline src/lib/clients/eCarmen/index.js]: Nooooo!!! This does not account for URL encoding rules. Instead, use `encodeURIComponent(universityDimName)`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

File: src/graphql/graduateExplorer/service.js (hunk 1 above)
Comment (hip1310) [reply src/graphql/graduateExplorer/service.js]: Fixed it. Thanks

File: src/graphql/graduateExplorer/typeDefs.js (hunk 2 above)
Comment (hip1310) [reply src/graphql/graduateExplorer/typeDefs.js]: Fixed it. Thanks

File: src/lib/clients/eCarmen/index.js (hunk 3 above)
Comment (hip1310) [reply src/lib/clients/eCarmen/index.js]: Fixed it. Thanks

**Pull request merged**